core/src/test/resources/sql/dummy.iq

```diff
@@ -20,58 +20,4 @@ values 1;
 EXPR$0
 1
 !ok
-
```

Comment:
Can you please add a comment here saying "do not modify this file, this is intended only for temporary debugging code"?

Reply:
Good suggestion, I've added a comment.